ConorMFinn/CO600-SeriousCyberSec
# Added questions to objects

**Description of work.**

Each question has 3 possible answers.
When the question has been answered the correct answer is given with feedback at the bottom.
10 questions have been allocated to objects around the house.

**To test:**

Start the game.
Find the objects with questions.
Answer the questions and check the feedback given.

Fixes #65
